app/assets/images/lecture-hall-building.geojson

```diff
@@ -0,0 +1,199 @@
+                    ],
+                    [
+                        13.13198000059,
+                        52.39425224785
+                    ]
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "indoor": "room",
+                "name": "HS1"
+            },
+            "geometry": {
+                "type": "LineString",
+                "coordinates": [
+                    [
+                        13.13250705466,
+                        52.39398791193
+                    ],
+                    [
+                        13.13280612096,
+                        52.39399936924
+                    ],
+                    [
+                        13.13278868660,
+                        52.39417695728
+                    ],
+                    [
+                        13.13248827919,
+                        52.39416468163
+                    ],
+                    [
+                        13.13250705466,
+                        52.39398791193
+                    ]
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "indoor": "room",
+                "name": "Foyer"
+            },
+            "geometry": {
+                "type": "LineString",
+                "coordinates": [
+                    [
+                        13.13240244851,
+                        52.39411721576
+                    ],
+                    [
+                        13.13221603498,
+                        52.39410903199
+                    ],
+                    [
+                        13.13217714295,
+                        52.39411066874
+                    ],
+                    [
+                        13.13213814970,
+                        52.39410924816
+                    ],
+                    [
+                        13.13214897975,
+                        52.39399691410
+                    ],
+                    [
+                        13.13211545214,
+                        52.39399445896
+                    ],
+                    [
+                        13.13211545214,
+                        52.39397072594
+                    ],
+                    [
+                        13.13250705466,
+                        52.39398791193
+                    ],
+                    [
+                        13.13248827919,
+                        52.39416468163
+                    ],
+                    [
+                        13.13239803057,
+                        52.39415877813
+                    ],
+                    [
+                        13.13240244851,
+                        52.39411721576
+                    ]
+                ]
+            }
+        },
+        {
+            "type": "Feature",
+            "properties": {
+                "indoor": "room",
+                "name": "HE5"
```

Comment:
The simple indoor tagging schema also uses areas and corridors for indoor mapping and not only rooms. Not sure if this is useful for us.